requirements-tutorials.txt

```diff
@@ -0,0 +1,11 @@
+# Requirements for the tutorials CI run that go beyond the others in `requirements-optional.txt`.
+# This may also include some requirements that are only in `requirements-dev.txt`, since those
+# aren't runtime dependencies or optionals of Terra.
+
+qiskit-ibmq-provider[visualization]
```

Comment:
This is needed for the tutorials? I'm a bit surprised by that because the tutorials shouldn't be using the ibmq provider for anything because they're run without credentials in CI.

Reply:
I'd just copied this over from what was being installed in this CI run before (except I think I added the `visualization` extra instead of additionally installing `matplotlib` separately).  I removed it in b558eb639.